ocs_ci/ocs/uninstall.py

```diff
@@ -212,14 +226,85 @@ def uninstall_ocs():
             .get("storageClassName")
         )
 
+    # step 6: delete storagecluster
+    cleanup_policy = (
+        storage_cluster.get()
+        .get("metadata")
+        .get("annotations")
+        .get("uninstall.ocs.openshift.io/cleanup-policy")
+    )
+
     log.info("Deleting storageCluster object")
     storage_cluster.delete(resource_name=constants.DEFAULT_CLUSTERNAME)
 
+    # step 7+9: check cleanup pods are compleated
+    storage_node_list = get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
+    if cleanup_policy == "delete":
+        # check clean up pods
+        cleanup_pods = [
+            pod for pod in get_all_pods() if "cluster-cleanup-job" in pod.name
+        ]
+        for pod in cleanup_pods:
+            if pod.get().get("status").get("phase") == "Succeeded":
+                log.info(f"Cleanup pod {pod.name} completed successfully ")
+            else:
+                log.error(f"Cleanup pod {pod.name} did not complete")
+        # confirm var/lib/rook is deleted
+        for node in storage_node_list:
+            try:
+                assert not ocp_obj.exec_oc_debug_cmd(
+                    node=node, cmd_list=["ls -l var/lib/rook"]
+                ), "OCS artificats were not deleted from nodes "
+            except CommandFailed:
+                pass
+
+        pass
+
+    # step 8: delete namespace
+    log.info("Deleting openshift-storage namespace")
+    ocp_obj.delete_project("openshift-storage")
+    ocp_obj.wait_for_delete("openshift-storage")
+    switch_to_project("default")
+
+    # step 10: TODO remove crypto from nodes
+    for node in storage_node_list:
+        log.info(f"removing encryption from {node}")
+        ocp_obj.exec_oc_debug_cmd(node=node, cmd_list=[])
+
+    # step 11: remove LSO
+    if lso_sc is not None:
+        uninstall_lso(lso_sc)  # TODO check lso func
+
+    # step 12: delete noobaa sc
+    log.info("deleting noobaa storage class")
+    noobaa_sc = ocp.OCP(kind=constants.STORAGECLASS)
+    noobaa_sc.delete(resource_name=constants.NOOBAA_SC)
+
+    # step 13: unlable storage nodes
+    node_objs = ocp.OCP(kind=constants.NODE).get().get("items")
+    label_nodes(nodes=node_objs, label=constants.OPERATOR_NODE_LABEL[:-3] + "-")
+    label_nodes(nodes=node_objs, label=constants.TOPOLOGY_ROOK_LABEL + "-")
+
+    # remove taint from nodes
+    taint_nodes(nodes=node_objs, taint_label=constants.OCS_TAINT + "-")
+
+    # step 14: delete remaining PVs
+    try:
+        rbd_pv = ocp.OCP(kind=constants.PV, resource_name="ocs-storagecluster-ceph-rbd")
+        fs_pv = ocp.OCP(kind=constants.PV, resource_name="ocs-storagecluster-cephfs")
```

Comment:
As well as for this https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L95